SU-SWS/jumpstart_ui
# D8CORE-387 Dynamic layouts

# READY FOR REVIEW

# Summary
- Removed all the layouts and create 3 basic ones with a configurable container option.

# Need Review By (Date)
- Turkey day

# Urgency
- medium

# Steps to Test
1. in the stack: `composer require su-sws/stanford_profile:dev-D8CORE-387-dynamic-layouts`
1. run a full site install `blt drupal:install`
1. edit the layout of a page or the content type
1. go to add a new section. verify you only get 3 options.
1. add an option and change its centered container.
1. verify it expands appropriately
1. view on of the interior pages.
1. verify there is no left sidebar
1. add a child page to the menu
1. verify it DOES have a left sidebar.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
